test/regressions/index.js

```diff
@@ -5,9 +5,12 @@ import webfontloader from 'webfontloader';
 import TestViewer from './TestViewer';
 
 // Get all the tests specifically written for preventing regressions.
-const requireRegression = require.context('./tests', true, /js$/);
+const requireRegression = require.context('./tests', true, /(js|ts|tsx)$/);
```

Comment:
A note for self: check that visually regression tests aren't run twice (.js + .tsx variants)

Reply:
This require context is for the standalone regression tests. They don't have a TS variant. For the docs we have a separate require context.